pkg/noun/manage.c

```diff
@@ -1833,35 +1855,15 @@ u3m_init(size_t len_i)
     exit(1);
   }
 
-  // map at fixed address.
-  //
-  {
-    void* map_v = mmap((void *)u3_Loom,
-                       len_i,
-                       (PROT_READ | PROT_WRITE),
-                       (MAP_ANON | MAP_FIXED | MAP_PRIVATE),
-                       -1, 0);
-
-    if ( -1 == (c3_ps)map_v ) {
-      map_v = mmap((void *)0,
-                   len_i,
-                   (PROT_READ | PROT_WRITE),
-                   (MAP_ANON | MAP_PRIVATE),
-                   -1, 0);
-
-      u3l_log("boot: mapping %zuMB failed", len_i >> 20);
-      u3l_log("see urbit.org/using/install/#about-swap-space"
-              " for adding swap space");
-      if ( -1 != (c3_ps)map_v ) {
-        u3l_log("if porting to a new platform, try U3_OS_LoomBase %p",
-                map_v);
-      }
-      exit(1);
-    }
-
-    u3C.wor_i = len_i >> 2;
-    u3l_log("loom: mapped %zuMB", len_i >> 20);
-  }
+  u3_pma = pma_load(u3_Loom,
```

Comment:
I see now. For some reason, I was under the impression that we had a file-backed home road, and then we were doing anonymous mappings for each sub-road. I think it was the `grows_down` parameter the got me.

A comment explaining that the PMA doesn't care about what happens with regards to roads - all it cares about is the state of the heap and stack right before sync, e.g. on the home road. I'm not sure where this comment should go, though, since it's about serf's usage of the PMA, and therefore shouldn't be in the PMA code itself.